md-commons/src/main/java/com/wipro/ats/bdre/md/app/AppStore.java

```diff
@@ -0,0 +1,17 @@
+package com.wipro.ats.bdre.md.app;
+import java.util.List;
+
+/**
+ * Created by cloudera on 3/10/16.
+ */
+public class AppStore {
+    public List<StoreJson> getApplicationList() {
+        return applicationList;
+    }
+
+    public void setApplicationList(List<StoreJson> applicationList) {
+        this.applicationList = applicationList;
+    }
+
+    List<StoreJson> applicationList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)